internal/proxyservice/ssh/proxy_service.go

```diff
@@ -0,0 +1,203 @@
+package ssh
```

Comment:
why not `ssh.go` for filename? I guess the question is mostly due to disambiguating this file and `proxyservice/proxy_service.go`

Reply:
i see your point. however, this is consistent with the other proxy services (tcp and http). i'd say that's a refactor for later

Reply:
😿  but makes sense